frontend/tests/features/collection.test.ts

```diff
@@ -20,9 +20,10 @@ const TEST_COLLECTION = {
 describe("Collection", () => {
   describeIfDeployed("Logged In Tests", () => {
     it("creates and deletes a collection", async () => {
+      const timestamp = Date.now();
       await login();
 
-      const collectionId = await createCollection();
+      const collectionId = await createCollection(timestamp.toString());
```

Comment:
prefer `String(timestamp)` for best practice, since values that don't have `toString()` method will throw runtime error E.g., `undefined.toString() // runtime error`